pyrobosim/examples/test_world.py

```diff
@@ -0,0 +1,55 @@
+"""
+Unit tests for core world modeling
+"""
+
+import pytest
+from pyrobosim.core.world import World
+from pyrobosim.core.world import Room
+
+
+class TestWorldModelling():
```

Comment:
Just for consistency, let's go with the American spelling throughout:

"Modelling" -> "Modeling"